tfjs-backend-webgpu/src/webgpu_util.ts

```diff
@@ -155,10 +155,7 @@ export function GPUBytesPerElement(dtype: DataType): number {
 }
 
 export function isWebGPUSupported(): boolean {
-  return ((typeof window !== 'undefined') ||
-          //@ts-ignore
-          (typeof WorkerGlobalScope !== 'undefined')) &&
-      !!navigator.gpu;
+  return !!globalThis.navigator.gpu;
```

Comment:
It seems that the functionality is equal before and after the change. Can you explain more why it can fix the Deno's NPM compatibility issue?

Reply:
I am ok with this change since it makes the code simpler. Just curious how it fix the issue you mentioned.

Reply:
Node doesnt have `window`, as such in our NPM compat we remove the object. However, we still have the global scope, so navigator is present, but not the `window` object itself.